### PR TITLE
Docs: add zeus, fix minor issues

### PR DESCRIPTION
**Pull request by BullHorn7**
Add zeus.md
Add zeus to modules.md
Fix standardization of other files

### CODE REVIEW

**Comment by BullHorn7** [comment]
Adding @SilentSpike for review

**Comment by esteldunedain** [comment]
:+1: As far as I'm concerned

File: documentation/missionmaker/modules.md [hunk 1]
```diff
@@ -560,3 +560,28 @@ Follow players. If set to false, loop will play sounds only nearby logic positio
...
+Spawn an eagle that follows the Zeus camera<br>
+`Default value: No`
+3. **Wind Sounds (Boolean)**<br>
+Play wind sounds when Zeus remote controls a unit<br>
+`Default value: No`
+4. **Ordnance Warning (Boolean)**<br>
+Play a radio warning when Zeus uses ordnance<br>
+`Default value: No`
+5. **Reveal Mines (Scalar)**<br>
+Should PAK be removed on usage?<br>
```
Comment (SilentSpike) [inline documentation/missionmaker/modules.md]: Little leftover copy-pasta it seems :spaghetti:

**Comment by SilentSpike** [comment]
All good other than that line :+1:

**Comment by BullHorn7** [comment]
Fixed. Thanks!

**Comment by BullHorn7** [comment]
I made a nearly identical PR, this is irrelevant. See https://github.com/acemod/ACE3/pull/1349

**Pull request closed without merging**